lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -230,16 +229,20 @@ if Enum.all?(
       {{:ok, remove_child: children_to_remove}, state}
     end
 
-    defp add_video_sink_children_if_no_video_left(children, tracks) do
-      sink_children = [:compositor, :encoder, :video_parser_out, :hls_sink_bin]
-
-      any_video_track_left? =
-        Enum.any?(tracks, fn {_id, track} ->
-          track.type == :video
-        end)
+    defp add_video_sink_children_if_no_video_left(children, []),
```

Comment:
We are using this function in line 227 to get children to remove. I think that the name should be changed as it is missleading